src/app/docs/inline-form/inline-form-docs.component.html

```diff
@@ -8,7 +8,48 @@
   </sky-docs-demo-page-summary>
 
   <sky-docs-demo>
-    See documentation for how to setup demos.
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+    >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-radio-group
+          heading="Default button configurations"
```

Comment:
Should this just say `Button configurations` ? What does the word "default" give us here?

Reply:
I didn't include "Custom" and didn't want to use the word "prefabrications". I can go with "Button configurations" if you're okay with excluding "Custom".